recipe/meta.yaml

```diff
@@ -112,264 +122,171 @@ outputs:
-        # boost-cpp is a pointer to libboost; include libboost so the linking metadata is correct.
-        - boost-cpp
-        - libboost
-        - double-conversion
-        - bzip2
-        # omnscidb Load-time UDF support calls clang++
-        - gxx_{{ target_platform }}
-        - libclang-cpp {{ llvm_version }}
-        - ncurses
-        - openjdk 8.*
-        - xz
-        - zlib
-        - {{ pin_subpackage('omniscidb-common',  max_pin='x.x.x') }}
-      run_constrained:
-        - cudatoolkit >=11.0  # [cuda_compiler_version not in (undefined, "None")]
-
-    test:
-      commands:
-        - test -f ${PREFIX}/{{ install_base }}/bin/omnisci_server
-        - test -f ${PREFIX}/{{ install_base }}/bin/omnisci_initdb
-        - test -f ${PREFIX}/{{ install_base }}/bin/omnisql
-        - test -f ${PREFIX}/{{ install_base }}/bin/KafkaImporter
-        - test -f ${PREFIX}/{{ install_base }}/bin/StreamImporter
-
-        # Verify the activation scripts are in-place.
-        {% for state in ["activate", "deactivate"] %}
-        - test -f "${PREFIX}/etc/conda/{{ state }}.d/{{ PKG_NAME }}_{{ state }}.sh"
-        {% endfor %}
-        # Try using the activation scripts.
-        - |
-          if [[ -x "$(command -v omnisql)" ]]
-          then
-            echo "Found omnisql in PATH"
-          else
-            echo "omnisql not found in PATH(=$PATH)" && exit 1
-          fi
-          source ${PREFIX}/etc/conda/deactivate.d/{{ PKG_NAME }}_deactivate.sh
-          if [[ -x "$(command -v omnisql)" ]]
-          then
-            echo "Unexpectedly found omnisql in PATH(=$PATH) after deactivation"  && exit 1
-          else
-            echo "omnisql not in PATH after deactivation"
-          fi
-          source ${PREFIX}/etc/conda/activate.d/{{ PKG_NAME }}_activate.sh
-        # Test installation
-        - omnisql -v
-        - which omnisci_initdb
-        - which omnisci_server
-        - |
-          if [ -x "$(command -v nvidia-smi)" ]; then  # [cuda_compiler_version not in (undefined, "None")]
-            mkdir data && omnisci_initdb data
-            omnisci_server --version
-            rm -rf data
-          fi                                          # [cuda_compiler_version not in (undefined, "None")]
-        # these are provided by omniscidb-common package:
-        - which omnisci_insert_sample_data
-        - which startomnisci
-
     about:
       home: https://www.omnisci.com/
       license: Apache-2.0
       license_family: APACHE
-      license_file: LICENSE.md
-      summary: The OmniSci database
-
+      license_file: omniscidb-common/share/doc/omnisci/LICENSE.md
+      summary: The OmniSci database common files.
       description: |
         OmniSciDB is an in-memory, column store, SQL relational database
         that was designed from the ground up to run on GPUs.
-
-        This recipe provides both CUDA enabled and CUDA disabled omniscidb
-        packages.
       doc_url: https://www.omnisci.com/docs/latest/
```

Comment:
There is a redirect in the `doc_url` to the following `URL`
https://docs.heavy.ai/